content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -392,7 +413,7 @@ The number of traces allowed to be submitted per second.
 If specified, adds all of the specified tags to all generated spans.
 
 `DD_TRACE_DEBUG`
-Enables or disables debug logging. Valid values are: `true` or `false`.<br>
+: Enables or disables debug logging. Valid values are: `true` or `false`.<br>
```

Comment:
```suggestion
: Enables or disables debug logging. Valid values include `true` or `false`.<br>
```

Reply:
I think that "Valid values include" implies there may be more values in addition to the ones we list here. That is not the case: these are _all_ the valid values for this setting.

Also, we already use "Valid values are:" in multiple places in this document.